docs/05-technical-reference/00-custom-resources/svls-01-function.md

```diff
@@ -145,13 +145,13 @@ This table lists all the possible parameters of a given resource together with t
 | **spec.resourceConfiguration.build.resources.limits.memory**         |       No       | Defines the maximum amount of memory available for the Job's Pod to use.      |
 | **spec.resourceConfiguration.build.resources.requests.cpu**          |       No       | Specifies the number of CPUs requested by the build Job's Pod to operate.       |
 | **spec.resourceConfiguration.build.resources.requests.memory**       |       No       | Specifies the amount of memory requested by the build Job's Pod to operate.               |
-| **spec.replicas**                             |       No       | Defines exact number of Function's Pods to run at a time. Can't be used at the same time with **spec.scaleConfig**. |
-| **spec.scaleConfig**                          |       No       | Defines minimum and maximum number of Function's Pods to run at a time. Can't be used at the same time with **spec.replicas**. |
+| **spec.replicas**                             |       No       | Defines exact number of Function's Pods to run at a time. If **spec.scaleConfig** field is configured, or if Function is targeted by external scaler, then the **spec.replicas** field is used by relevant HorizontalPodAutoscaler to control the number of active replicas. |
```

Comment:
```suggestion
| **spec.replicas**                             |       No       | Defines the exact number of Function's Pods to run at a time. If **spec.scaleConfig** is configured, or if Function is targeted by an external scaler, then the **spec.replicas** field is used by the relevant HorizontalPodAutoscaler to control the number of active replicas. |
```

Reply:
fixed in .md and .go files and regenerated